packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,682 @@
+---
+title: Factor Profiles
+category: management
+---
+
+# Factor Profiles API
+
+The Okta Factor Profiles API enables an Administrator configure which factor profiles are available to use for multi-factor authentication.
```

Comment:
> "enables an Administrator configure"
Change to "enables an Administrator _to_ configure"

Reply:
There also seems to be an inconsistency in capitalization. Why is "Factor Profiles" in title case in some places and lower case in others?

Or is it because "The Okta Factor Profiles API" as the whole is in title case and the rest is not required to be?

Reply:
If "Factor Profiles" are a specific thing in Okta, then I think they should be capitalized throughout. That being said, I wasn't going to do that sort of review until we'd had the sign off from the technical side of team.